contracts/examples/lottery-esdt/src/lottery.rs

```diff
@@ -84,10 +86,11 @@ pub trait Lottery {
     ) {
         require!(!lottery_name.is_empty(), "Name can't be empty!");
 
+        let max_tickets = MAX_TICKETS.try_into().unwrap();
```

Comment:
`unwrap` adds some panic logic, so we try to avoid it as much as possible. I think it's best if you leave the constant as u32 and do `as usize` where you need it as usize.

Reply:
Best way would be to change u32 to usize everywhere, so no conversions needed.
Otherwise, `as` is the way indeed

Reply:
I need to use it as a generic `ArrayVec<usize, MAX_TICKETS as usize>`
this is why I thought of making it usize 

apparently it doesn't allow me to do this